### ISSUE TITLE
I pray and hope Cosmic-Station will become the next best PS2 Emulator In Android History

### ISSUE THREAD

**Issue by Ottorock** [darkluar/cosmic-station#6]

**Comment by Miguel-hrvs** [comment]
If the dev needs testers I'm here to help as much as I can.

**Comment by shadergz** [comment]
Someday, actually we're in a very initial stage by now, nothing works or anything could be seen on screen by now...

**Comment by Miguel-hrvs** [comment]
I imagine that now there's not much to test. Good luck with your emu! One question, why did you use dobiestation as reference instead of play! or pcsx2? Is performance related? The emu will be hle like play! or it will require bios files like pcsx2?

**Comment by shadergz** [comment]
Basically, it's about the level of complexity. It's much easier for me to understand the (quick and dirty) code created by the Dobie group than to try to understand the feats of a much more sophisticated code like that of PCSX2, but in the future, yes, PCSX2 will become the base project for this one.

**Comment by shadergz** [comment]
And no, everyone will need a BIOS

**Comment by shadergz** [comment]
The next step now is to improve the interface and bring the first graphics to the screen

**Comment by Miguel-hrvs** [comment]
Thanks for your responses, I'll use my ps2 7004 dumped bios when you have something for testing, good luck :)

**Comment by omagadsamsa** [comment]
it will never be like play! it would be better if it weren't for netherx2

**Comment by Miguel-hrvs** [comment]
Yes, I think that netherx2 should be working in an open source project instead of patching a closed source one.
